forms/jqm/element.js

```diff
@@ -68,9 +71,13 @@ define(function (require) {
     renderLabel: function () {
       if (!this.$label) {
         this.$label = $('<label class="ui-input-text"></label>');
+      }
+
+      if (!this.$el.find(this.$label).length) {
```

Comment:
Another change you have here removes the label from both the DOM and from the View's references. Doesn't that change alone cover this issue?